library/java/src/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -50,7 +56,9 @@ String resolveTemplate(String templateYAML) {
             .replace("{{ dns_failure_refresh_rate_seconds_max }}",
                      String.format("%s", dnsFailureRefreshSecondsMax))
             .replace("{{ stats_flush_interval_seconds }}", String.format("%s", statsFlushSeconds))
-            .replace("{{ device_os }}", "Android");
+            .replace("{{ device_os }}", "Android")
```

Comment:
We should move forward with these changes but something that we will need to figure out when we revisit our ergononmics:
These properties leak the Android abstraction onto the `EnvoyConfiguration` :(

Ideally, we should have a separate configuration for Android and Java libraries

Reply:
Sure, this could be moved to be a param that gets injected by the client builder. Won't fix in this PR as it was already here.

Reply:
Nope. Don't fix in this PR :)